packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
```

Comment:
I think all source files need it, why would dart files not need it? (@hixie)

Can you file an issue about adding this to `flutter create` at https://github.com/flutter/flutter/issues/new?